src/coreclr/nativeaot/Runtime/inc/CommonTypes.h

```diff
@@ -6,10 +6,35 @@
 
 #include <cstddef>
 #include <cstdint>
-#include <cstdlib>
-#include <new>
+#include <stdlib.h>
+#include <stdio.h>
+
+#ifdef TARGET_WINDOWS
 
+#include <new>
 using std::nothrow;
+
+#define VIRTUAL_IMPL = 0;
```

Comment:
Nit: I think "pure" is an important to describe what this does. I would call this `PURE_VIRTUAL_IMPL` or just `PURE_VIRTUAL`.

Reply:
The one in #else branch is not pure, so I went with a neutral name.

Reply:
It acts like a pure virtual impl in both cases. I think the name should describe what it does, not how it is implemented.